flytekit/clis/sdk_in_container/run.py

```diff
@@ -763,6 +767,57 @@ def get_command(self, ctx, name):
         )
 
 
+class YamlFileReadingCommand(click.RichCommand):
+    def __init__(self, name: str, params: typing.List[click.Option], help: str, callback: typing.Callable = None):
+        params.append(
+            click.Option(
+                ["--flyte-inputs-file"],
```

Comment:
At this point, we are already running `pyflyte run ...`. Can the option be called `--inputs-file`?